bin/plotting/pycbc_page_foundmissed

```diff
@@ -95,11 +113,18 @@ try:
                              f['injections/inclination'][:]))
     eff_dists = numpy.array(eff_dists).T
 
-    # "Decisive" distance is the second smallest effective distance
+    # "Decisive" distance for coincidences is the second smallest
+    # effective distance
     dvals['decisive_distance'] = numpy.sort(eff_dists)[:,1]
     dvals['dec_chirp_distance'] = \
         pycbc.pnutils.chirp_distance(dvals['decisive_distance'],
                                      vals['mchirp'])
+    # When including single-detector triggers in the analysis, the
+    # "decisive" distance is be the minimum distance
```

Comment:
So .. I think the use of 'decisive' only makes sense in the coinc context as it was thought of originally as the distance that controls whether a coinc is made or not. Here I would just write eg "When single-detector triggers are included in the analysis, minimum distance is the relevant quantity for injection efficiency"

Reply:
Done